etc/docker-scripts/bootstrapper.py

```diff
@@ -239,7 +233,7 @@ def execute(self) -> None:
         print("Executing Python Script : {} ==> {}".format(python_script, python_script_output))
         try:
             with open(python_script_output, "w") as log_file:
-                subprocess.check_call(['python', python_script], stdout=log_file, stderr=subprocess.STDOUT)
+                subprocess.run(['python', python_script], stdout=log_file, stderr=subprocess.STDOUT)
```

Comment:
Does this need `check=True`?

Reply:
I have added `check=True` but it didn't make much difference...